.github/workflows/build.yml

```diff
@@ -0,0 +1,335 @@
+          go-version: ${{ needs.get-go-version.outputs.go-version }}
+    
+      - name: Build dependencies
+        run: make deps
+      
+      - name: Build generate-all
+        run: make generate-all
+      
+      - name: Setup node and yarn
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+          cache: 'yarn'
+          cache-dependency-path: 'ui/yarn.lock'
+      
+      - name: Install Yarn
+        run: |
+          npm install -g yarn
+      
+      - name: UI Build
+        run: |
+          cd ui
+          yarn install --network-timeout 300000
+          yarn --verbose run build
+          cd ..
+      
+      - name: Build static assets
+        run: make static-assets  
+
+      - name: Build
+        env:
+          GOOS: ${{ matrix.goos }}
+          GOARCH: ${{ matrix.goarch }}
+          LD_FLAGS: ${{ needs.set-ld-flags.outputs.ldflags }}
+          CGO_ENABLED: 1
+        run: |
+          mkdir -p dist out
+          go build -o dist/ \
+          -ldflags "${{ env.LD_FLAGS }}" \
+          -tags "${{ env.GO_TAGS }}"
+          zip -r -j out/${{ env.PKG_NAME }}_${{ needs.get-product-version.outputs.product-version }}_${{ matrix.goos }}_${{ matrix.goarch }}.zip dist/
+      - uses: actions/upload-artifact@v2
+        with:
+          name: ${{ env.PKG_NAME }}_${{ needs.get-product-version.outputs.product-version }}_${{ matrix.goos }}_${{ matrix.goarch }}.zip
+          path: out/${{ env.PKG_NAME }}_${{ needs.get-product-version.outputs.product-version }}_${{ matrix.goos }}_${{ matrix.goarch }}.zip
+   
+  build-linux:
+    needs: [ get-go-version, get-product-version, set-ld-flags ]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        goos: [ linux ]
+        goarch: [ "arm", "arm64", "386", "amd64" ]
+      fail-fast: true
+    
+    name: Go ${{ needs.get-go-version.outputs.go-version }} ${{ matrix.goos }} ${{ matrix.goarch }} build
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ needs.get-go-version.outputs.go-version }}
+      
+      - name: Build dependencies
+        run: make deps
+      
+      - name: Build generate-all
+        run: make generate-all
+      
+      - name: Setup node and yarn
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+          cache: 'yarn'
+          cache-dependency-path: 'ui/yarn.lock'
+      
+      - name: Install Yarn
+        run: |
+          npm install -g yarn
+      
+      - name: UI Build
+        run: |
+          cd ui
+          yarn install --network-timeout 300000
+          yarn --verbose run build
+          cd ..
+
+      - name: Build static assets
+        run: make static-assets
+
+      - name: Build
+        env:
+          GOOS: ${{ matrix.goos }}
+          GOARCH: ${{ matrix.goarch }}
+          LD_FLAGS: ${{ needs.set-ld-flags.outputs.ldflags }}
+          CGO_ENABLED: 0
```

Comment:
We actually need `CGO` to be enabled in all of our builds. I think the only time we disable `CGO` is when cross-compiling a Linux binary from Mac.